### ISSUE TITLE
Whalebird icon Ubuntu

### ISSUE THREAD

**Issue by RotFarm** [h3poteto/whalebird-desktop#4225]
The image used for the program icon is squished at the sides? It looks like this in the app launcher and in the "show applications" area in Ubuntu 20.04
![walbirgd](https://user-images.githubusercontent.com/10239722/226347999-675f6441-7395-493a-a363-7d3548b389ea.png)

**Comment by h3poteto** [comment]
- How to install Whalebird?
- Which version of Whalebird are you using?
- What happened to the original icon files? You can find it in `/usr/share/icons`.

**Comment by RotFarm** [comment]
I'm using the 5.0.2 snap on Ubuntu 20.04

**Comment by FiSHYsolutions** [comment]
i am using xubuntu 22.04.2 lts (xfce v4.16.1) and the systray icon of whalebird (snap) is not available since an upgrade from v4.7.4 to v5.0.1 which i did on march 10th. i am now using v5.0.2 but it isn't fixed.

![Bildschirmfoto_2023-03-29_18-54-28](https://user-images.githubusercontent.com/222112/228612145-ba285ccd-4606-4ba5-bbb7-a2d9b4b43af6.png)

**Comment by h3poteto** [comment]
@FiSHYsolutions Are you talking about the tray icon? Probably, the tray icon issue is another cause.

**Comment by FiSHYsolutions** [comment]
yes, i am talking about the missing tray icon. and i recognized broken symlinks pointing to an empty `/snap/whalebird/current/data-dir/icons/`. maybe there is a correlation, maybe not. anyway: we shouldn't have broken symlinks at all.

**Comment by h3poteto** [comment]
Ok, could you please create another issue?

**Comment by FiSHYsolutions** [comment]
same issue in #3824 closed without fix but was referenced to #3831 which was closed with a reference to #3832

**Comment by h3poteto** [comment]
Thanks

**Comment by FiSHYsolutions** [comment]
we don't have a systray-icon anymore (not even a buggy one) after updating the snap to v5.0.4
